### PR TITLE
Add speakup support to kernel

### PR DESCRIPTION
**Pull request by thcrain-msft**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Add speakup and uinput as kernel modules. These are needed to implement text-to-speech accessibility in the ISO.

###### Change Log  

- On both architectures:  `CONFIG_ACCESSIBILITY=y`, `CONFIG_SPEAKUP=m`, `CONFIG_SPEAKUP_SYNTH_SOFT=m`
- On x64: `CONFIG_INPUT_UINPUT=y` (already present on aarch64)
- Add `drivers-accessibility` subpackage to kernel

###### Does this affect the toolchain?  

**YES**

###### Test Methodology

- Local build of ISO image with new kernel options

### CODE REVIEW

File: SPECS/kernel/kernel.spec [hunk 1]
```diff
@@ -452,6 +467,9 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %endif
 
 %changelog
+* Mon Feb 22 2021 Thomas Crain <thcrain@microsoft.com> - 5.10.13.1-2
+- Add configs for speakup and uinput drivers
```
Comment (PawelWMS) [inline SPECS/kernel/kernel.spec:471]: Suggestion: maybe also mention the new subpackage name somewhere in the changelog.

**Review by PawelWMS — APPROVED** [review]
Approving, but please consult `config` file changes with Chris as well.

File: SPECS/kernel/kernel.spec (hunk 1 above)
Comment (thcrain-msft) [reply SPECS/kernel/kernel.spec:471]: Fixed, thanks!

**Review by jslobodzian — APPROVED** [review]
:shipit:

File: SPECS/kernel/config [hunk 2]
```diff
@@ -2886,7 +2886,7 @@ CONFIG_INPUT_MISC=y
 # CONFIG_INPUT_POWERMATE is not set
```
Comment (christopherco) [inline SPECS/kernel/config:2886]: Since the `config` and `config_aarch64` files were changed, the hashes in kernel.signature.json need to be updated too.

**Pull request merged**